README.md

```diff
@@ -22,6 +22,10 @@ To get a local copy up and running follow these simple example steps.
 
 git clone https://github.com/ernestkorir/my-portfolio.git
 
```

Comment:
[Optional]  You can use `justify-content: space-between`  to reduce the right space of the burguer menu between the border of the screen 

![image](https://user-images.githubusercontent.com/95243996/193552934-ded56086-dd9c-46a8-b023-027423e84f94.png)